src/twisted/conch/test/test_ckeygen.py

```diff
@@ -63,6 +64,46 @@ def setUp(self):
         self.patch(sys, 'stdout', self.stdout)
 
 
+
+    def test_runecdsa(self):
+        sys.argv = ['ckeygen', '-t', 'ecdsa', '-f', 'test_ecdsa', '--no-passphrase']
+        run()
```

Comment:
So I suspect these tests are causing your breakage. This is changing the entire runtime environment of the test, which is a bit alarming. It would almost certainly be much better to execute ckeygen in a subprocess rather than do this.